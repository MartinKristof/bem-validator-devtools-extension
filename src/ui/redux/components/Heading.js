import React from 'react';
import PropTypes from 'prop-types';

const Heading = ({ children }) => (
  <div>
    <h4>{children}</h4>
  </div>
);

Heading.propTypes = {
  children: PropTypes.string.isRequired,
};

export default Heading;
