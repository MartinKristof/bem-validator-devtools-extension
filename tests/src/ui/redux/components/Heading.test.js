import React from 'react';
import { shallow } from 'enzyme';
import Heading from '../../../../../src/ui/redux/components/Heading';

describe('<Heading />', () => {
  describe('#render', () => {
    it('should render without side-effects', () => {
      const node = shallow(<Heading>Foo</Heading>);

      expect(node).toMatchSnapshot();
    });
  });
});
