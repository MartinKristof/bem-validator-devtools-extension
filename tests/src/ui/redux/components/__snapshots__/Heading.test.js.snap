// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading /> #render should render without side-effects 1`] = `
<div>
  <h4>
    Foo
  </h4>
</div>
`;
